pam/authmodeselection.go

```diff
@@ -72,21 +72,32 @@ func (m authModeSelectionModel) Update(msg tea.Msg) (authModeSelectionModel, tea
 		m.availableAuthModes = msg.authModes
 
 		var allAuthModes []list.Item
+		var firtAuthModeID string
```

Comment:
```suggestion
		var firstAuthModeID string
```

Plus all the refactoring related to the change...

Reply:
Oupsss, well spotted! I took the liberty to amend the commit directly due to the size :)